components/sup/src/manager/mod.rs

```diff
@@ -494,6 +494,9 @@ impl Manager {
             let es_config =
                 cfg.event_stream_config
                    .expect("Config should be present if the EventStream feature is enabled");
+            // TODO: @afiune We could use a public crate to gather the hostname
```

Comment:
I'm not a fan of TODOs in checked-in code. If it's worth doing, file and issue (and optionally reference it here), but TODOs that live only in code tend to either be fixed right away or stick around forever and don't get included in work prioritization and planning.

Reply:
Agree! 💯 I am working on it as we type 😸

Reply:
If we can't implement the crate I mentioned, I will definitely remove this TODO. Thanks @baumanj

Reply:
We should almost certainly be able to use https://crates.io/crates/hostname. I'd file an issue, with a comment referencing it at https://github.com/habitat-sh/habitat/blob/master/components/core/src/os/net/unix.rs#L20 and https://github.com/habitat-sh/habitat/blob/master/components/core/src/os/net/windows.rs#L22, but remove the TODO here, since it's not where the change needs to be made.

Reply:
@baumanj that crate is abandoned.

Reply:
I was thinking that we might need to implement the C function `getnameinfo()` instead of `gethostname()`. I was going to work on that next but I’ll difer to you @baumanj or @christophermaier to agree or disagree.